app/build.gradle

```diff
@@ -61,8 +64,12 @@ dependencies {
     testImplementation "androidx.room:room-testing:$room_version"
 
     implementation "com.chargemap.compose:numberpicker:1.0.3"
+    implementation "androidx.compose.runtime:runtime-livedata:1.3.2"
```

Comment:
@grittaweisheit `livedata` is a new dependency that needs to be installed. Just resync the project 👍